util/include/lru_cache.hpp

```diff
@@ -37,8 +37,8 @@ class LruCache {
         map_.erase(keys_.back());
         keys_.pop_back();
       }
-      keys_.push_front(key);
-      map_.insert({key, {value, keys_.begin()}});
+      keys_.push_front(std::move(key));
+      map_.insert({key, {std::move(value), keys_.begin()}});
```

Comment:
The `key` variable is used after move on line 41. I think there's no way around copying the key at least once.

Reply:
thanks for the catch!